Arrowgene.Ddon.GameServer/BazaarManager.cs

```diff
@@ -211,7 +211,8 @@ private List<BazaarExhibition> GetSoldExhibitionsByCharacter(Character character
         private uint calculateProceeds(CDataBazaarItemBaseInfo itemBaseInfo)
         {
             uint totalPrice = itemBaseInfo.Num*itemBaseInfo.Price;
-            return (uint)Math.Clamp(totalPrice - totalPrice*TAXES, 0, uint.MaxValue);
+            uint taxDeduction = (uint)(totalPrice * TAXES);
+            return Math.Clamp(totalPrice - taxDeduction, 0, uint.MaxValue);
```

Comment:
Is `TAXES` some value between 0 and 1? If not, we should check to make sure `totalPrice > taxDeduction`.

Reply:
TAXES is 0.05; it's left as `private static readonly double TAXES = 0.05;` in `BazaarManager`, but since the clientside tax calculation assumes that value without any input from the server, its practically unchangeable.